### ISSUE TITLE
[FEATURE] Sinerider Monorepo

### ISSUE THREAD

**Issue by camdan-me** [hackclub/sinerider#342]
### Description

Move the other sinerider-related repos over here to unify all of the sinerider frontend and backend systems.

**Comment by alhardwarehyde** [comment]
Potential alternative solution: https://git-scm.com/book/en/v2/Git-Tools-Submodules

**Comment by camdan-me** [comment]
Yeah, they're just kinda painful. I feel like submodules would be a barrier to entry

**Comment by GalaxyGamingBoy** [comment]
Yea def, submodules would be a great idea.
Don't know about the deployment tho.

**Comment by alhardwarehyde** [comment]
Moving this to post ship for now
